packages/iconset/stories/icons-demo.ts

```diff
@@ -249,6 +249,16 @@ export class IconsDemo extends SpectrumElement {
             });
         }
 
+        // Use a Set to remove duplicates that may get added because of the same icon name in both versionsa
+        const uniqueIcons = new Set();
+        matchingIcons = matchingIcons.filter((icon) => {
```

Comment:
For example, when on the `spectrum` system, it will show by default
> Showing 891 of 1248 available icons.

and in `spectrum-two`:

> Showing 357 of 1248 available icons.

I am not sure it is very transparent to customers why the other ones are missing. Should we instead show how many matching icons there are per system?
So in S1 it would be

> Showing 891 of 891 available icons.

and S2

> Showing 357 of 357 available icons.

Reply:
done!